include/deal.II/base/symmetric_tensor.h

```diff
@@ -2338,87 +2338,84 @@ namespace internal
 {
   namespace SymmetricTensorImplementation
   {
-    namespace
```

Comment:
I would name it ‘internal’ which suggests that users shouldn’t use it

Reply:
@davydden But this is already `internal::SymmetricTensorImplementation`...

Reply:
Oh, then it’s fine. Did not notice this from phone